src/codelabs/async-await.md

```diff
@@ -192,7 +192,7 @@ function.
 First, add the `async` keyword before the function body:
 
 {% prettify dart %}
-main() [!async!] {
+void main() [!async!] {
 {% endprettify %}
 
 {{site.alert.note}}
```

Comment:
This note should be dropped now that `main` has a declared return type. (This codelab focuses on async programming, IMHO, we don't need to mention optional types.)